tests/generate-erc20-permit.test.ts

```diff
@@ -80,7 +80,7 @@ describe("generateErc20PermitSignature", () => {
     (context.adapters.supabase.wallet.getWalletByUserId as jest.Mock).mockReturnValue(null);
 
     await expect(async () => {
-      await generateErc20PermitSignature(SPENDER, amount, context);
+      await generateErc20PermitSignature(context, SPENDER, amount);
```

Comment:
Why did you move this back to the first parameter? I recall earlier in the conversation that we agreed optional parameters should be the last?

Reply:
Relates to the conversation [here](https://github.com/ubiquibot/permit-generation/pull/7#discussion_r1555450639)
Parameters were not optional, just overloaded.